tests/test_haydn_op20.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import music21
+
+from mirdata.datasets import haydn_op20
+from tests.test_utils import run_track_tests
+
+
+def test_track():
+    default_trackid = "0"
+    data_home = "tests/resources/mir_datasets/haydn_op20"
+    dataset = haydn_op20.Dataset(data_home)
+    track = dataset.track(default_trackid)
+
+    expected_attributes = {
+        "humdrum_annotated_path": "tests/resources/mir_datasets/haydn_op20/op20n1-01.hrm",
+        "title": "op20n1-01",
+        "track_id": "0",
+    }
+
+    expected_property_types = {
+        "duration": int,
+        "chords": list,
+        "roman_numerals": list,
+        "keys": list,
+        "score": music21.stream.Score,
+        "midi_path": str,
+    }
+    run_track_tests(track, expected_attributes, expected_property_types)
+
+
+def test_to_jam():
+    data_home = "tests/resources/mir_datasets/haydn_op20"
+    dataset = haydn_op20.Dataset(data_home)
+    track = dataset.track("0")
+    jam = track.to_jams()
+    assert jam["file_metadata"]["title"] == "op20n1-01", "title does not match expected"
+    assert jam["file_metadata"]["duration"] == 12152, "duration does not match expected"
+    assert (
+        jam["sandbox"]["humdrum_annotated_path"]
+        == "tests/resources/mir_datasets/haydn_op20/op20n1-01.hrm"
+    ), "duration does not match expected"
+    assert (
+        jam["sandbox"]["midi_path"]
+        == "tests/resources/mir_datasets/haydn_op20/op20n1-01.midi"
+    ), "duration does not match expected"
+    assert type(jam["sandbox"]["chords"]) == list
```

Comment:
Check for correct loading of data in the `jams_converter`

Reply:
done